src/widgets/MeterPanel.cpp

```diff
@@ -1126,8 +1126,8 @@ void MeterPanel::OnMeterUpdate(wxTimerEvent & WXUNUSED(event))
 
    if (numChanges > 0) {
       #ifdef EXPERIMENTAL_AUTOMATED_INPUT_LEVEL_ADJUSTMENT
-         if (gAudioIO->AILAIsActive() && mIsInput && !discarded) {
-            gAudioIO->AILAProcess(maxPeak);
+         if (AudioIO::Get()->AILAIsActive() && mIsInput && !discarded) {
```

Comment:
It seems that the only place `EXPERIMENTAL_AUTOMATED_INPUT_LEVEL_ADJUSTMENT` should be used in such approach is `AILAIsActive`.